roles/common/defaults/main.yml

```diff
@@ -120,6 +120,7 @@ common:
         - python-virtualenv
         - python-mysqldb
         - python-jinja2
+        - PyYAML
```

Comment:
apparently on rhel is also PyYAML
[root@ds0104 ~]# rpm -qa | grep -i yaml
PyYAML-3.10-11.el7.x86_64
libyaml-0.1.4-11.el7_0.x86_64